konnectd/Makefile

```diff
@@ -88,10 +88,9 @@ generate: assets
 
 # TODO find a docker container with go and yarn so we can properly build assets in ci
 #assets: assets/identifier/static assets/identifier/index.html
-assets: assets/identifier/static
+assets: static-assets
```

Comment:
This needs to be run always, not only if it doesn't exist. When running `yarn build` it builds a placeholder logo for dev env. We are overwriting it with this with oC logo.